include/config/SkUserConfig.h

```diff
@@ -6,7 +6,6 @@
  * found in the LICENSE file.
  */
 
-
```

Comment:
Can the additional endl in this file be removed for easier upgrades in the future?